Sources/TensorFlow/Operators/Math.swift

```diff
@@ -644,6 +644,23 @@ public func rsqrt<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
     Raw.rsqrt(x)
 }
 
+/// Returns the cosine similarity between `x` and `y`.
+@differentiable(wrt: (x, y))
+public func cosineSimilarity<Scalar: TensorFlowFloatingPoint>(
+    _ x: Tensor<Scalar>, _ y: Tensor<Scalar>
+) -> Tensor<Scalar> {
+    return (x * y).sum() /
```

Comment:
I think if you remove the `return` keyword the whole body can fit in one line.

```suggestion
    (x * y).sum() / (sqrt(x.squared().sum()) * sqrt(y.squared().sum()))
```